modules/custom/lua/new_player_linkshell.lua

```diff
@@ -0,0 +1,15 @@
+-----------------------------------
+-- Set if you want new players to get a linkshell
+-----------------------------------
+require("modules/module_utils")
+require("scripts/globals/player")
+-----------------------------------
+local m = Module:new("new_player_linkshell")
+
+m:addOverride("xi.player.charCreate", function(player)
+    local lsName = "InsertLsNameHere" -- Name of linkshell
+    player:addLinkpearl(lsName, true)
+    super(player)
+end)
```

Comment:
When you do this in this order, do you get the LS welcome message when you log in? 

If not:

I would maybe shuffle this around so the super is called first, and then use the result of addLinkpearl:

Super()

If player:addLinkpearl() then
    Send system_3 message saying you've joined that LS
End

Reply:
yes you get the lsmes
![ffxi_2022 06 11_02 43 36](https://user-images.githubusercontent.com/6879206/173176794-d8036755-4da1-4de2-bde6-43ced7c2b628.jpg)